install/deploy_daemon.sh

```diff
@@ -2,14 +2,19 @@
 
 DIR=${1:-.}
 USER=${2:-user}
-DOCKER_ACC=${3:-fwnetworking}
+DOCKER_ACC=${3:-"localhost:5000"}
+YAML_FILE="daemon.deploy.yaml"
 
 # Build the daemon image
 docker image build -t $DOCKER_ACC/dropletd:latest -f $DIR/mgmt/etc/docker/daemon.Dockerfile $DIR
-if [[ "$USER" == "dev" ]]; then
+if [[ "$USER" == "dev" || "$USER" == "final" ]]; then
```

Comment:
what is user final?

Reply:
User final will push to the fwnetworking dockerhub repo.